webpack/scenes/FlatpakRemotes/CreateEdit/EditFlatpakRemotesModal.js

```diff
@@ -13,18 +13,25 @@ const EditFlatpakModal = ({ show, setIsOpen }) => (
     onClose={() => { setIsOpen(false); }}
     appendTo={document.body}
   >
-    <FlatpakRemotesForm setModalOpen={setIsOpen} />
+    <FlatpakRemotesForm setModalOpen={setIsOpen} remoteData={remoteData} />
   </Modal>
 );
 
 EditFlatpakModal.propTypes = {
   show: PropTypes.bool,
   setIsOpen: PropTypes.func,
+  remoteData: PropTypes.shape({
+    name: PropTypes.string.isRequired,
```

Comment:
You can mention id in the prop validation here so that when we send out the update request, we know we have an ID to send it to..